README.md

```diff
@@ -1 +1,305 @@
+[![LSIO CI](https://img.shields.io/badge/dynamic/yaml?color=94398d&labelColor=555555&logoColor=ffffff&style=for-the-badge&label=CI&query=CI&url=https%3A%2F%2Fci-tests.linuxserver.io%2Flinuxserver%2Ffail2ban%2Flatest%2Fci-status.yml)](https://ci-tests.linuxserver.io/linuxserver/fail2ban/latest/index.html)
+
+[Fail2ban](http://www.fail2ban.org/) is a daemon to ban hosts that cause multiple authentication errors.
+
+[![fail2ban](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/fail2ban-logo.png)](http://www.fail2ban.org/)
+
+## Supported Architectures
+
+We utilise the docker manifest for multi-platform awareness. More information is available from docker [here](https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-2.md#manifest-list) and our announcement [here](https://blog.linuxserver.io/2019/02/21/the-lsio-pipeline-project/).
+
+Simply pulling `lscr.io/linuxserver/fail2ban:latest` should retrieve the correct image for your arch, but you can also pull specific arch images via tags.
+
+The architectures supported by this image are:
+
+| Architecture | Available | Tag |
+| :----: | :----: | ---- |
+| x86-64 | ✅ | amd64-\<version tag\> |
+| arm64 | ✅ | arm64v8-\<version tag\> |
+| armhf| ✅ | arm32v7-\<version tag\> |
+
+## Application Setup
+
+This container is designed to allow fail2ban to function at the host level, as well as at the docker container level.
+If you are running applications on the host, you will need to set the `chain` to `INPUT` in the jail for that application.
+All jails require the ability to read the application log files.
+We recommend mounting each application's logs as a volume to the container (illustrated by the optional volumes in our documentation).
+
+### Configuration Files
+
+On first run, the container will create a number of folders and files in `/config`. The default configurations for fail2ban are all disabled by default.
+
+Please refer to the [Configuration README](https://github.com/linuxserver/fail2ban-confs/blob/master/README.md), which can be viewed in our repository, or in your config folder at `/config/fail2ban/README.md`.
+
+### Remote Logs
+
+The `/remotelogs` path is designed to act as a parent for all log files you would like fail2ban to be able to use.
+Each log file should be mounted in a subfolder underneath `/remotelogs`, ex:
+- `/remotelogs/nginx/` would mount a folder containing the nginx logs to the container
+- `/remotelogs/unificontroller/server.log` would mount a single file for the unifi controller logs to the container
+
+## Usage
+
+Here are some example snippets to help you get started creating a container.
+
+### docker-compose (recommended, [click here for more info](https://docs.linuxserver.io/general/docker-compose))
+
+```yaml
+---
+version: "2.1"
+services:
+  fail2ban:
+    image: lscr.io/linuxserver/fail2ban:latest
+    container_name: fail2ban
```

Comment:
Should the example use host networking by default?
I can't imagine this being too useful without it

Reply:
Good catch, and yes, you are correct. I thought I had it with https://github.com/linuxserver/docker-fail2ban/blob/initial/readme-vars.yml#L21

I'll go dig through Jenkins to find what actually does it.

Reply:
Fixed in https://github.com/linuxserver/docker-fail2ban/pull/1/commits/b2e566deae964c6ee10a0f3fe0e57e808b4610d4